src/contract.ts

```diff
@@ -358,38 +361,55 @@ export function buildTypeClasses(desc: CompileResult | ContractDescription): Rec
 
   const structClasses = buildStructsClass(desc);
```

Comment:
Used deprecated method??

Reply:
buildStructsClass is deprecated  , but it still work, buildTypeClasses is more powerful. we suggest users to using buildTypeClasses